README.md

```diff
@@ -14,33 +14,156 @@ The system uses a number of open source projects to work properly:
 * [ElizaBOT-JS] - Javascript implementation of the ELIZA specification by Weizenbaum, 1966. Special thanks to Landsteiner!
 
 
-# Installation:
+## Installation:
 
-TEXA requires [Go Lang](https://golang.org/)  v1.7+ to run.
+### Go:
 
-Install the dependencies, devDependencies and start the server.
+TEXA is written in [Go](http://golang.org) v1.7+. If you don't have a Go
+development environment, please [set one up](http://golang.org/doc/code.html).
+
+### Redis: 
+
+Texa using the [texalib]
```

Comment:
How about we reform the sentence as:

 "TEXA uses Redis to store all the mathematical data used by the `texalib` package, depended by the TEXA API. You can install [redis](https://redis.io/) and learn more about redis [here](https://redis.io/topics/quickstart#redis-quick-start)." ?